src/screens/Learn/Faq/FaqStructuredData.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { Fragment } from 'react';
+import {
+  JSONLD,
+  Question,
+  Answer,
+  GenericCollection,
+  // @ts-ignore: getting error for this import :(
+} from 'react-structured-data';
+import {
+  faqContent,
+  FaqSection,
+  Question as QuestionItem,
+} from 'cms-content/learn';
+
+const FaqStructuredData: React.FC = () => {
+  const { sections } = faqContent;
+  return (
+    <JSONLD>
+      <GenericCollection type="FAQPage">
+        {sections.forEach((section: FaqSection) => {
```

Comment:
We need to use `map` instead, because [`forEach`](http://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) always returns `undefined` (it just iterates over the elements of the array). That's why the `script` element is almost empty in the DOM.

![image](https://user-images.githubusercontent.com/114084/103572433-65c3a380-4e81-11eb-9d4a-3160cb3dc36c.png)